### PR TITLE
v0.2.4 release

### PR DESCRIPTION
**Pull request by ryankurte**
Updated changelog and crate version, once approved / merged i will publish it to crates.io.

### CODE REVIEW

**Comment by eldruin** [comment]
@ryankurte Could you pull #220 before the release as well?

**Comment by ryankurte** [comment]
cc. @therealprof i think this is good to go

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+
+## [v0.2.4] - 2020-06-10
+
+### Added
+
+
```
Comment (therealprof) [inline CHANGELOG.md:20]: Empty on purpose?

Comment (ryankurte) [reply CHANGELOG.md:20]: yeah we haven't added anything new afaik, maintenance only.

Comment (therealprof) [reply CHANGELOG.md:20]: Not a big fan of an empty section but could live with that.

**Comment by therealprof** [comment]
Probably need to adjust the master branch, too.

**Pull request merged**